scripts/deploy.ts

```diff
@@ -1,8 +1,11 @@
 // TODO There is a eslint configuration error that needs to be fixed
 import { keccak256, toUtf8Bytes, hexZeroPad } from 'ethers/lib/utils'
-import { ethers } from 'hardhat'
+import { ethers, network } from 'hardhat'
+import getChanges, { ContractsChange } from './get-changes'
 import { waitForTransactionMined } from './utils'
 
+let changes: ContractsChange[] = ['ENS', 'BMT', 'DAPP_REGISTRY']
```

Comment:
I would initialize it in the `main` function since it is closer to its corresponding logic and mutable variables is not a good design on the first level of scope.